### PR TITLE
[Keyboard] Add info.json to O4L5x12

### PR DESCRIPTION
**Pull request by BenRoe**
## Description
Add a info.json to the keycapsss/o4l5x12 keyboard to support the qmk configurator.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/keycapsss/o4l_5x12/info.json [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+                "y": 4
+            }, {
+                "x": 3,
+                "y": 4
+            }, {
+                "x": 4,
+                "y": 4
+            }, {
+                "x": 5,
+                "y": 4
```
Comment (noroadsleft) [inline keyboards/keycapsss/o4l_5x12/info.json]:
```suggestion
                "y": 4,
                "w": 2
```

File: keyboards/keycapsss/o4l_5x12/info.json [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+            }, {
+                "x": 2,
+                "y": 4
+            }, {
+                "x": 3,
+                "y": 4
+            }, {
+                "x": 4,
+                "y": 4
```
Comment (noroadsleft) [inline keyboards/keycapsss/o4l_5x12/info.json]:
```suggestion
                "y": 4,
                "w": 2
```

File: keyboards/keycapsss/o4l_5x12/info.json [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+            }, {
+                "x": 6,
+                "y": 4
```
Comment (noroadsleft) [inline keyboards/keycapsss/o4l_5x12/info.json]:
```suggestion
                "y": 4,
                "w": 2
```

**Comment by noroadsleft** [comment]
Travis CI errors are due to Community Layout feature. All reference keymaps (`default`, `2x2u` and `default_ortho_5x12`) compile without errors.

**Comment by BenRoe** [comment]
> I personally find the file a bit unwieldy when formatted this way, but that's more a nitpick, and I wouldn't reject the PR based on that. It's just quite a lot of extra lines.

You are right. It was the auto formatting from VSC. Changed it to one key per row.

**Comment by BenRoe** [comment]
> That looks much better. `key_count` key/value pairs would be nice for the other two layouts, but again, that's my personal preference.

Done

**Comment by drashna** [comment]
Travis CI error is unrelated to this keyboard (issues with community layouts).

Thanks!

**Pull request merged**